fragments/fields/README.md

```diff
@@ -1,19 +1,15 @@
-# STAC API - Fields Fragment
+  # STAC API - Fields Fragment
 
 - **OpenAPI specification:** [openapi.yaml](openapi.yaml)
 - **Fragment [Maturity Classification](../../extensions.md#extension-maturity):** Pilot
 - **Dependents:**
   - [Item Search](../../item-search)
 
```

Comment:
Maybe start here with a introduction what the fragment is meant to achieve. It starts with too many details (e.g. content-types).

Reply:
Will do.